src/fold/composites/columns.py

```diff
@@ -20,24 +20,23 @@ class PerColumnEnsemble(Composite):
     ----------
     pipeline: Pipeline
         Pipeline (list of Pipeline) to ensemble
-    models_already_cloned: bool
-        For internal use. It determines if the pipeline has been copied or not.
 
-    Returns
-    ----------
-    X: pd.DataFrame
-        Ensemble of outputs of passed in pipelines.
-    y: pd.Series
-        Target passed along.
     """
 
     properties = Composite.Properties()
     models_already_cloned = False
 
-    def __init__(self, pipeline: Pipeline, models_already_cloned: bool = False) -> None:
+    def __init__(self, pipeline: Pipeline) -> None:
```

Comment:
hmm, this looks wrong, we should probably use Pipelines type here

Reply:
sorry, not true, this needs to be a single Pipeline, need to change the descipriont